simplified-app-simplye/src/main/assets/Accounts.json

```diff
@@ -60,6 +60,130 @@
     "mainColor": "#497049"
   },
   {
+    "id": 7,
```

Comment:
There should be no changes to Accounts.json in this PR. Go ahead and reset the file it to the master branch. Before doing that, ensure that Accounts.json residing in the Simplified-iOS repo is up-to-date in regards to anything that may have been changing. This is to ensure predictability between platforms in our testing. Next time I'll get out in front of this a little better!

Reply:
Sorry I take this comment back. It looks like the branch was just really out of date. Still be sure that Accounts.json on iOS is the way you want it, however. And double check the newer keys for password attributes that I mentioned above.